src/cpu/pred/BranchPredictor.py

```diff
@@ -93,6 +127,20 @@ class SimpleBTB(BranchTargetBuffer):
     instShiftAmt = Param.Unsigned(
         Parent.instShiftAmt, "Number of bits to shift instructions by"
     )
+    associativity = Param.Unsigned(1, "BTB associativity")
+    size = Param.MemorySize("16384", "Size of BTB")
```

Comment:
Is this a leftover from the initial version?

Reply:
Indeed, removed the line. I will squash the commits if you think this is OK. Thanks!

Reply:
Yes feel free to squash

Reply:
Done.